types/controllers/index.d.ts

```diff
@@ -177,6 +177,19 @@ export interface IDoughnutControllerDatasetOptions
   extends IControllerDatasetOptions,
     ScriptableAndArrayOptions<IArcOptions>,
     ScriptableAndArrayOptions<IArcHoverOptions> {
+  
+  /**
+   * Sweep to allow arcs to cover.
+   * @default 2 * Math.PI
```

Comment:
do you think it would be good to take the leap towards degrees with options in these controllers too?

Reply:
Yes, we should probably swap radians to degrees to make it simpler. I'll look at updating this

Reply:
I updated both to degrees, and moved the `0` rotation point from the right side to the top to match polar area and radar charts.